scripts/col-repl.js

```diff
@@ -28,11 +28,12 @@ var repl = require('repl');
 
 var DB = require('col-core/lib/db');
 
-/**
- * Load Collabosphere environment and start REPL
- */
 console.log('Loading Collabosphere environment...');
 
+/**
+ * Load Collabosphere environment and start an instance of Node's REPL (read-eval-print loop), 
+ an interactive interpreter that evaluates input line by line and displays evaluation results.
```

Comment:
Also, can you add an `@see` at the bottom of this block pointing to the Node REPL repo ?